### ISSUE TITLE
addSeries into setSize into redraw lets the latest bar overlap others in stacked bar charts

### ISSUE THREAD

**Issue by Bloodmallet** [highcharts/highcharts#13752]
#### Expected behaviour
The usual behaviour of each bar reserving its own space.

#### Actual behaviour
The latest series element is placed on top of the other bars, without "pushing" them to the right.

#### Live demo with steps to reproduce
[js-fiddle](https://jsfiddle.net/zwjh26yk/)
Click the button in the fiddle to see the overlap. The comment in the fiddle shows the correct behaviour, if a different order of commands is used.

#### Product version

Currently latest stable version. 8.1.2

#### Affected browser(s)
Chrome Version 83.0.4103.106 (Official Build) (64-bit)

**Comment by pawelfus** [comment]
Thank you for reporting the issue @Bloodmallet 

**Workaround:**
Plugin: 
```js
(function(H) {
  H.wrap(
    H.Series.prototype,
    'setStackedPoints',
    function(proceed) {
      if (!this.stackKey && this.options.stacking) {
        this.chart.getStacks();
      }

      return proceed.apply(this, Array.prototype.slice.call(arguments, 1))
    }
  );
})(Highcharts);
```
Demo: https://jsfiddle.net/BlackLabel/aqhtjL29/

*Internal note:*
For some reason `series.stackKey` is not set in `chart.setSize()`.

**Comment by khlieng** [comment]
Fixed on master (#14414).
